Homeworks/homework_6/aircrafts/cargo.py

```diff
@@ -0,0 +1,18 @@
+from Homeworks.homework_6.aircrafts.aircraft import Aircraft
+
+
+class Cargo(Aircraft):
+    width_range = [5, 20]
+    speed_range = [400, 600]
+    landing_distance_range = [2000, 3000]
+
+    def __init__(self, speed, aircraft_id, width=None, num_of_seats=2, landing_distance=0):
+        super().__init__(speed=speed,
+                         aircraft_id=aircraft_id,
+                         width=width,
+                         num_of_seats=num_of_seats,
+                         landing_distance=landing_distance)
```

Comment:
you don't need to overload init if you don't have functionality to extend the basic initializer